lib/site_prism/element_container.rb

```diff
@@ -154,7 +160,7 @@ def create_nonexistence_checker(element_name, *find_args)
         method_name = "has_no_#{element_name}?"
         create_helper_method(method_name, *find_args) do
           define_method(method_name) do |*runtime_args|
-            wait_time = SitePrism.use_implicit_waits ? Capybara.default_max_wait_time : 0
+            wait_time = SitePrism.use_implicit_waits ? max_wait_time : 0
```

Comment:
How about we extract this line out to new Line 11. And switch it from a ternary to if/else?